client/src/pages/DiscoverFeed.css

```diff
@@ -47,10 +62,36 @@
 }
 
 @media screen and (min-width: 767px) {
+  .second-wrapper {
+    right: 40vh;
+  }
+
   .img-div {
     margin: -113px;
+    position: relative;
   }
 
+  .img-div:hover .view-closet,
+  .img-div:focus .view-closet {
+    opacity: 1;
+    visibility: visible;
+    transform: scale(1);
+  }
+
+  .view-closet {
+      font-size: 100px;
+      white-space: nowrap;
+      position: absolute;
+     right: 0px;
+     top: 20%;
+      z-index: 100;
+      color: white;
+      overflow-x: visible;
+      opacity: 0;
+      transform: scale(1.1) ;
+      transition: opacity .2s, transform .2s;
```

Comment:
You might also try a text-shadow property so the white text shows up over the white background. Something like:
```css
text-shadow: 1px 1px 1px black;
```
![Uploading Screen Shot 2022-04-11 at 6.28.07 PM.png…]()